tasks/functions/functions.js

```diff
@@ -3,44 +3,59 @@
  *
  */
 function add(a, b) {
+ return a + b;
 
 }
+console.log(add(-1, -2));
 
 /**
  * write function that return first and last name of given object
  * properties firstName and lastName
  * {
- *    firstName: "John"
- *    lastName: "Dou"
+ *    {firstName: "John", lastName: "Dou"}
+ *    lastName: "Dou" 
  * }
  */
-function getFullName(object) {
+function getFullName(sum) {
+    return sum.firstName + ' ' + sum.lastName;
+    } 
+console.log(getFullName({firstName: "John", lastName: "Dou"}));
 
-}
 
 /**
  * write fuction that checks is number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+    return n % 2>0 ? true : false;
 }
+console.log(isOdd(2));
 
 /**
  * write function that return shortest of words in given array
  * e.g ["one", "two", "three"] should return one
  */
+// TOODDDDDDDDDDDDDOOOOO 1
 function getShortest(wordArray) {
-
+	let shortelement = wordArray[0];
+for (let i = 1; i < wordArray.length; i++) {
+		if (wordArray[i].length < shortelement.length) {
+			shortelement = wordArray[i]}
 }
+ return shortelement;
+}
+console.log(getShortest(["one", "two", "three", "x"]));
 
 /**
  * write function that returns word google with given numbers of "o" symbols
  * e.g getGoogle(5) should return "gooooogle"
  */
-function getGoogle(n) {
+let letter = 'o'
```

Comment:
please define the variable inside the function

Reply:
Done